### ISSUE TITLE
Console can not connect to RPC service

### ISSUE THREAD

**Issue by mtsbh** [pal1000/Realtek-UAD-generic#118]
Hi, thanks for your work! 

I did everything in the tutorial and it worked on my previous computer, but I'm with a new one now and did exactly (I think) like the last time, but I'm not sure that everything is okay. First of all, the console can't connect to the RPC service. I tried to uninstall and download from Microsoft Store, but I get an error when trying to download from there. 

Also, I'm not sure that last time it was named in sounds (Speakers Realtek High Definition Audio). I think it was else.

![image](https://user-images.githubusercontent.com/108582199/193271218-533da1cd-3a81-4053-b1b1-804fb135ede3.png)

**Comment by pal1000** [comment]
Yeah, you may be running a legacy HDA driver which means you have to go through [this](https://github.com/pal1000/Realtek-UAD-generic#switching-from-other-realtek-hd-audio-drivers). It's a drag, but it is what it is.

**Comment by mtsbh** [comment]
Yeah, I've done that. It was the same after this. Maybe I should run the process all over again. Thank you
